components/dashboard/SideBar.js

```diff
@@ -24,6 +24,18 @@ const Sticky = styled.div`
   z-index: 10;
 `;
 
+const menuWrapperMobileMixin = `
```

Comment:
Must use `css` for these cases

```suggestion
const menuWrapperMobileMixin = css`
```